package.json

```diff
@@ -53,13 +53,13 @@
     "@fortawesome/fontawesome-free": "^6.6.0",
     "@getalby/bitcoin-connect": "^3.6.2",
     "@scure/base": "^1.1.7",
-    "@welshman/app": "^0.0.7",
+    "@welshman/app": "file:../welshman/packages/app",
```

Comment:
You can use `npm link` to install these from a local file.

- Run `npm link` in `welshman/packages/app`
- Run `npm link @welshman/app` in `coracle`

If you need to link multiple packages you'll need to list all package names in the `npm link` command, otherwise it won't take. If it doesn't work, remove node_modules, npm install, then link again.

Reply:
Ugh, sorry, I thought I'd amended the commit to remove these, I'd tried using the link command but it wasn't taking for the reasons you mention. I'll try again...